src/connectors/components/WalletListConnector.js

```diff
@@ -0,0 +1,21 @@
+// @flow
```

Comment:
This file should be called WalletListModalConnector

Reply:
Fixed